Recurly/Errors/ApiErrors.cs

```diff
@@ -128,96 +128,112 @@ public class BadRequest : ApiError
         public BadRequest(string message) : base(message) { }
         public BadRequest(string message, Exception inner) : base(message, inner) { }
     }
+    [ExcludeFromCodeCoverage]
```

Comment:
@cainj we generally also add these tags on generated code as this isn't really the responsibility of the library. Depends on how much "code" is actually there.

Reply:
Yes, this is actually restoring the `ExcludeFromCodeCoverage` that was mistakenly removed in a previous change in the generator.